src/bin/agent/CommandStart.ts

```diff
@@ -23,6 +23,8 @@ class CommandStart extends CommandPolykey {
     this.addOption(binOptions.clientPort);
     this.addOption(binOptions.ingressHost);
     this.addOption(binOptions.ingressPort);
+    this.addOption(binOptions.connTimeoutTime);
+    this.addOption(binOptions.seedNodes);
     this.addOption(binOptions.background);
```

Comment:
Add an additional one called `this.addOption(binOptions.network);` to specify the default network to connect to.

The relationship between `--seed-nodes` and `--network` is this:

```
# use whatever is in the default mainnet
pk agent start --network=mainnet
# the --seed-nodes completely overrides the network parameter
pk agent start --network=mainnet --seed-nodes=<nodeid>@<host>:<port>;<nodeid>@<host>:<port>
# this inherits the default seed nodes
pk agent start --network=mainnet --seed-nodes=<nodeid>@<host>:<port>;<nodeid>@<host>:<port>;<default>
```

I think the `<default>` is more self-explanator over `<seed-nodes>`.

Reply:
Done.